### ISSUE TITLE
Low number of BUSCOs in my genome

### ISSUE THREAD

**Issue by Joscett** [nextgenusfs/funannotate#402]
Hi, 

I'm using the funannotate V1.7.2 release. My genome is about ~3Gb in size, it is a plant genome.
I ran the funannotate predict with transcriptome and ESTs for --transcript_evidences, besides the uniprot proteins. The genome has 196 single and complete orhtologs in the BUSCO analysis, therefore the EVM step is not running.
Is there any way to use the other complete and duplicated orthologos founded by BUSCO? 

Thanks a lot.

I running the script in a cluster,

Here my parameters

funannotate predict -i Vpla_mask.fasta -o VPLA_V2 -s "Vanilla planifolia" --strain vpla_pasa --name Vplanifolia --protein_evidence ../../EVIDENCE_VAN/uniprot_Viridiplantae.faa ../../EVIDENCE_VAN/Vapla_orchidstra_CDS.faa --transcript_evidence ../../EVIDENCE_VAN/TranscComb2Fil_V2.fna ../../EVIDENCE_VAN/Vapla_orchidstra_CDS.fna ../../EVIDENCE_VAN/VAN_EST.fna --repeats2evm --busco_db embryophyta --min_intronlen 10 --max_intronlen 50000 --organism other --cpus 20

here the log files:

[04:03 PM]: OS: linux2, 32 cores, ~ 265 GB RAM. Python: 2.7.15
[04:03 PM]: Running funannotate v1.7.2
[04:03 PM]: Parsed training data, run ab-initio gene predictors as follows:
[04:04 PM]: Loading genome assembly and parsing soft-masked repetitive sequences
[04:10 PM]: Genome loaded: 18,249 scaffolds; 3,257,934,503 bp; 10.85% repeats masked
[04:10 PM]: Aligning transcript evidence to genome with minimap2
[04:22 PM]: Found 154,768 alignments, wrote GFF3 and Augustus hints to file
[04:22 PM]: Mapping 61,913 proteins to genome using diamond and exonerate
[05:19 PM]: Found 6,757,696 preliminary alignments --> aligning with exonerate
[11:15 AM]: Exonerate finished: found 187,424 alignments
[11:23 AM]: Running GeneMark-ES on assembly
[11:11 PM]: 314,821 predictions from GeneMark
[11:11 PM]: Running BUSCO to find conserved gene models for training ab-initio predictors
[05:17 AM]: 170 valid BUSCO predictions found, now formatting for EVM
[06:14 AM]: Running EVM commands with 19 CPUs
[07:22 AM]: Converting to GFF3 and collecting all EVM results
[07:26 AM]: 196 total gene models from EVM, now validating with BUSCO HMM search
[07:33 AM]: 196 BUSCO predictions validated
[07:33 AM]: Not enough gene models 196 to train Augustus (200 required), exiting

**Comment by nextgenusfs** [comment]
This is likely a failed BUSCO run as there should be 1440 models in the embryophyta BUSCO database -- almost always this is due to an Augustus installation.  There should be a `busco.log` file -- check there to see what happened in the BUSCO run. 

```
$ funannotate database --show-buscos
-----------------------------
BUSCO DB tree: (# of models)
-----------------------------
eukaryota (303)
	metazoa (978)
		nematoda (982)
		arthropoda (1066)
			insecta (1658)
			endopterygota (2442)
			hymenoptera (4415)
			diptera (2799)
		vertebrata (2586)
			actinopterygii (4584)
			tetrapoda (3950)
			aves (4915)
			mammalia (4104)
		euarchontoglires (6192)
			laurasiatheria (6253)
	fungi (290)
		dikarya (1312)
			ascomycota (1315)
				pezizomycotina (3156)
					eurotiomycetes (4046)
					sordariomycetes (3725)
					saccharomycetes (1759)
						saccharomycetales (1711)
			basidiomycota (1335)
		microsporidia (518)
	embryophyta (1440)
	protists (215)
		alveolata_stramenophiles (234)
```

**Comment by Joscett** [comment]
I will try with  this option --busco_seed_species. Thanks a lot.

Just to clear my doubt, 

the file generated by BUSCO is the next one:

# BUSCO version is: 2.0 
# The lineage dataset is: embryophyta_odb9 (Creation date: 2016-02-13, number of species: 30, number of BUSCOs: 1440)
# To reproduce this run: python /opt/intel/anaconda3/envs/funannote/lib/python2.7/site-packages/funannotate/aux_scripts/funannotate-BUSCO2.py -i /LUSTRE/usuario/jcetz/SAGARPA/VANILLA_V6/FUNannotate/VPLANIFOLIA/predict_misc/genome.softmasked.fa -o vanilla_planifolia_vpla_pasa -l /data/secuencias/funannotate_db/embryophyta/ -m genome -c 20 -sp anidulans
#
# Summarized benchmarking in BUSCO notation for file /LUSTRE/usuario/jcetz/SAGARPA/VANILLA_V6/FUNannotate/VPLANIFOLIA/predict_misc/genome.softmasked.fa
# BUSCO was run in mode: genome

        C:76.6%[S:11.8%,D:64.8%],F:5.3%,M:18.1%,n:1440

        1103    Complete BUSCOs (C)
        170     Complete and single-copy BUSCOs (S)
        933     Complete and duplicated BUSCOs (D)
        76      Fragmented BUSCOs (F)
        261     Missing BUSCOs (M)
        1440    Total BUSCO groups searched

In this output file, the number of Complete and single is 170. This orthologs are taken for training, but i have other 933  (completed and duplicated) not used in the training step. Is there a way fun annotate predict can use this orthologs?

**Comment by nextgenusfs** [comment]
Ah. Try to increase the —ploidy option.
